pkg/daemon/daemon.go

```diff
@@ -314,8 +314,9 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 	dn.pluginManager.OnPTPConfigChange(nodeProfile)
 
 	ptp_processes := []string{
-		ts2phcProcessName,
+		// make sure ptp4l is running first to get pmc ready
```

Comment:
Will this cause issues with ptp4l propagating bad times before ts2phc sets the time to the correct time in a GM config?